### PR TITLE
SDA-3589 (Remove unwanted targets and add extra files)

### PR DESCRIPTION
**Pull request by KiranNiranjan**
## Description
- Add extra files (screen snippet, cloud9 & screen share indicator frame)
- Remove nsis and dmg build targets

### CODE REVIEW

**Review by keerthi16 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -41,14 +41,15 @@
   },
   "build": {
     "appId": "com.symphony.electron_desktop",
-    "uninstallDisplayName": "${productName}",
     "artifactName": "${productName}-${version}-${os}.${ext}",
     "asar": true,
     "asarUnpack": "**/*.node",
     "nsis": {
```
Comment (sbenmoussati) [inline package.json:47]: [Question]: we still have an nsis build entry. However, nsis target is being removed. Is there a specific reason for that?

File: package.json [hunk 2]
```diff
@@ -41,14 +41,15 @@
...
       "perMachine": true,
```
Comment (sbenmoussati) [inline package.json:48]: [Question] does it mean we would need admin rights to perform an auto update?

File: package.json (hunk 1 above)
Comment (KiranNiranjan) [reply package.json:47]: @sbenmoussati instead of removing `zip` I had removed nsis here.

File: package.json (hunk 2 above)
Comment (KiranNiranjan) [reply package.json]: Admin permission is required only if the user is installing in the program files directory

`perMachine` and `oneClick` will be **false** and the user will be shown an option to choose the install directory

**Review by sbenmoussati — APPROVED** [review]

**Pull request merged**